packaging/install/drain-cleaner/kubernetes/webhook-certificates/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
```

Comment:
Should we check for the existence of `cfssl` and exit with a message if it's missing?